features/admin.applications.v1/pages/applications.tsx

```diff
@@ -176,35 +184,64 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
         error: myAccountStatusFetchRequestError
     } = useMyAccountStatus(!isSubOrg && applicationConfig?.advancedConfigurations?.showMyAccountStatus);
 
+    /**
+     * Set the application id for My Account.
+     */
+    useEffect(() => {
+        if (myAccountApplicationData?.applications?.length === 1) {
```

Comment:
What if some end user had created an app named "My Account"? then the applications array length would be 2 and this condition will return false, wouldn't it?

Reply:
We cannot create an application with the name `My Account` in the console. Hence this issue could not happen.